base/task.jl

```diff
@@ -330,18 +330,49 @@ end
 Wrap an expression in a [`Task`](@ref) and add it to the local machine's scheduler queue.
```

Comment:
Need to add documentation of interpolation

Reply:
Done. Thanks.